go-fuzz-build/main.go

```diff
@@ -194,9 +213,20 @@ func buildInstrumentedBinary(pkg string, deps map[string]bool, lits map[Literal]
 		// Cross-compilation is not implemented.
 		copyDir(filepath.Join(GOROOT, "pkg", runtime.GOOS+"_"+runtime.GOARCH), filepath.Join(workdir, "goroot", "pkg", runtime.GOOS+"_"+runtime.GOARCH), true, nil)
 	}
+
+	wg := sync.WaitGroup{}
```

Comment:
`var wg sync.WaitGroup` please as more standard.
In one codebase I counted 2598 uses of `var wg sync.WaitGroup` vs 1 for `wg := sync.WaitGroup{}`.